jsk_pr2_robot/jsk_pr2_startup/CMakeLists.txt

```diff
@@ -20,3 +20,17 @@ endforeach()
 
 install(FILES install_pr1040_description.sh jsk_pr2.machine plugin.xml startup.app
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+
+# Create email config for PR2
+macro(configure_email_topic_yaml rname)
+  cmake_host_system_information(RESULT PR2_NAME QUERY HOSTNAME)
+  if (${PR2_NAME} MATCHES ^${rname})
```

Comment:
```suggestion
  if (${PR2_NAME} STREQUAL "pr1012" OR ${PR2_NAME} STREQUAL "pr1040") 
```

Reply:
btw, do we need to check if the machine's hostname?
we can compile yaml file in any machine, I think.

Reply:
I started to think we do not have to check hostname. 
Update in b012e94